src/plugins/kibana_legacy/public/font_awesome/font_awesome.scss

```diff
@@ -0,0 +1,32 @@
+@font-face {
+  font-family: 'FontAwesome';
+  src: url('~font-awesome/fonts/fontawesome-webfont.eot?v=4.7.0');
+  src: url('~font-awesome/fonts/fontawesome-webfont.eot?#iefix&v=4.7.0') format('embedded-opentype'),
+  url('~font-awesome/fonts/fontawesome-webfont.woff2?v=4.7.0') format('woff2'),
+  url('~font-awesome/fonts/fontawesome-webfont.woff?v=4.7.0') format('woff'),
+  url('~font-awesome/fonts/fontawesome-webfont.ttf?v=4.7.0') format('truetype'),
+  url('~font-awesome/fonts/fontawesome-webfont.svg?v=4.7.0#fontawesomeregular') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@import "font-awesome/scss/variables";
+@import "font-awesome/scss/mixins";
+@import "font-awesome/scss/core";
+@import "font-awesome/scss/larger";
+@import "font-awesome/scss/fixed-width";
+@import "font-awesome/scss/list";
+@import "font-awesome/scss/bordered-pulled";
+@import "font-awesome/scss/animated";
+@import "font-awesome/scss/rotated-flipped";
+@import "font-awesome/scss/stacked";
```

Comment:
Do we really need all the variations of FA? I don't think we use any other classes besides the base `.fa .fa-iconType`. Including these all will unnecessarily bloat the bundle.

Reply:
That's a good point, I just replicated what the old implementation was doing (it also imported all of these variants), but you are probably right we can scope them down. I will look into it.